static/css/help.less

```diff
@@ -6,41 +6,7 @@
     border-radius: 4px;
     padding: 2px 4px;
   }
-  dl.horizontal {
```

Comment:
I kind of want to… not do this. `btn` is used evvvvvvvvvvvvvvverywhere, and I'm not that confident that I know how to scope this kind of thing correctly so I don't break something.

I imagine it's all going to get re-written in either 2.14 or 4.0 anyway, so I think I'm going to try to avoid this one ;-)